### DISCUSSION TITLE
Ventura required

### DISCUSSION THREAD

**Discussion by rubyredrick** [dataFude-app/dataFude#9] [Beta feedback]
I tried to install via TestFlight but it says MacOS 13 is required. although the web site says MacOS 12. I'm still waiting on a few crucial apps to be compatible before I can upgrade.

**Comment by icanzilb** [comment]
Hey @rubyredrick, the app is currently macOS13 + only — I'm sorry about the confusion. Can you please point me to the spot where it says it's macOS 12 so I can correct that?
